pkg/generate/config/config.go

```diff
@@ -155,6 +155,9 @@ type AdditionalFieldConfig struct {
 	// Input shape in the Operation identified by OperationID that
 	// we will take as our additional spec/status field.
 	MemberName string `json:"member_name"`
+	// RenameMemberName refers to the name that will be used instead of
+	// MemberName in the status.
+	RenameMemberName string `json:"rename_member_name,omitempty"`
```

Comment:
Let's call these fields `SourceName` and `TargetName`. I think that most explicitly describes what the values represent. Thoughts?

Reply:
Done updated